### ISSUE TITLE
Error generating styles-m.css from Less library

### ISSUE THREAD

**Issue by vpietri** [magento/magento2#1391]
Hi,

Hi have an error on loading styles-m.css after removing var/ and pub/static/ directories content.

* Magento: ver. 0.74.0-beta13
* oyejorge/less.php: v1.7.0.3

Html content for url
*http://..../pub/static/frontend/Magento/blank/fr_FR/css/styles-m.css*
is
```
Fatal error:  Allowed memory size of 134217728 bytes exhausted at /opt/phpfarm/src/php-5.5.22/ext/standard/array.c:2277 (tried to allocate 72 bytes) in <b>/var/www/magento2/ce-2/vendor/oyejorge/less.php/lib/Less/Tree/Ruleset.php on line 637
```

**Comment by orlangur** [comment]
> Fatal error:  Allowed memory size of 134217728 bytes exhausted 

This is much less than default 768 Mb memory limit: https://github.com/magento/magento2/blob/develop/.htaccess#L39

Increasing memory limit to recommended value could solve your problem.

**Comment by vpietri** [comment]
You're right.

I'm running with Apache and php fastCGI. In this configuration php initialisation from .htacces is not allowed.

Many Thanks.

**Comment by slavvka** [comment]
Hi @vpietri. Thank you for your report. 
The issue has been fixed in magento/magento2#29207 by @jmonteros422 in 2.4-develop branch
Related commit(s):
 - [a3e9fea451c76359a10bf84df95f506f3e1fded3](https://github.com/magento/magento2/commit/a3e9fea451c76359a10bf84df95f506f3e1fded3)
 - [7c2511aa47b5addaa49a335c8c93ee61f572ab88](https://github.com/magento/magento2/commit/7c2511aa47b5addaa49a335c8c93ee61f572ab88)
 - [d81d26224481bd25ac1522af41a4aed1065a2ecb](https://github.com/magento/magento2/commit/d81d26224481bd25ac1522af41a4aed1065a2ecb)
 - [fe0be1ddd881d6d645b01adb0280ffd6f3538e64](https://github.com/magento/magento2/commit/fe0be1ddd881d6d645b01adb0280ffd6f3538e64)
 - [659071cd533f4d7a61256a899828c2dfc54ac171](https://github.com/magento/magento2/commit/659071cd533f4d7a61256a899828c2dfc54ac171)
 - [b166cf57b72cd50c0485a9f4cc5fa0203f26b2c0](https://github.com/magento/magento2/commit/b166cf57b72cd50c0485a9f4cc5fa0203f26b2c0)
 - [cc4c5d0f217874044ffd63fb0941213b54730d19](https://github.com/magento/magento2/commit/cc4c5d0f217874044ffd63fb0941213b54730d19)
 - [958fee38d7aa17b58131b01b4f4e6f77e7afd60f](https://github.com/magento/magento2/commit/958fee38d7aa17b58131b01b4f4e6f77e7afd60f)

The fix will be available with the upcoming 2.4.1 release.
